just/bluefin-tools.just

```diff
@@ -272,3 +272,58 @@ k8s-dev-tools:
     #!/usr/bin/bash
     echo "Adding Kubernetes command line tools..."
     brew bundle --file /usr/share/ublue-os/homebrew/kubernetes.Brewfile
+
+# Set up command-not-found for Homebrew
+setup-brew-not-found ACTION="":
+    #!/usr/bin/env bash
+    source /usr/lib/ujust/ujust.sh
+
+    OPTION={{ ACTION }}
+    if [ "$OPTION" == "help" ]; then
+        echo "Usage: ujust setup-brew-not-found <option>"
+        echo "  <option>: Specify the quick option to skip the prompt"
+        echo "  Use 'enable' to select Enable Brew Not Found"
+        echo "  Use 'disable' to select Disable Brew Not Found"
+        exit 0
+    elif [ "$OPTION" == "" ]; then
+        echo "${bold}Brew command-not-found Setup${normal}"
+        OPTION=$(Choose "Enable Brew command-not-found" "Disable Brew command-not-found")
+    fi
+
+    set -euo pipefail
+
+    BREW_BINARY=/home/linuxbrew/.linuxbrew/bin/brew
+    BREW_ROOT=${HOMEBREW_REPOSITORY:-$($BREW_BINARY --repository)}
+    if ! $BREW_BINARY -h > /dev/null; then
+      echo "Make sure Homebrew is first. Check journalctl -e -u brew-setup.service"
+      exit
+    fi
+
+    if [[ "${OPTION,,}" =~ ^enable ]]; then
+        $BREW_BINARY tap homebrew/command-not-found
+        pkexec tee /etc/profile.d/brew-command-not-found.sh > /dev/null <<EOF
+    # Check for interactive bash or zsh and that we haven't already been sourced
+    if [[ -d /home/linuxbrew/.linuxbrew && \$- == *i* && BREW_COMMAND_NOT_FOUND != 1 ]] ; then
+      HB_CNF_HANDLER="${BREW_ROOT}/Library/Taps/homebrew/homebrew-command-not-found/handler.sh"
+      [ -f "\$HB_CNF_HANDLER" ] && source "\$HB_CNF_HANDLER"
+      export BREW_COMMAND_NOT_FOUND=1
+    fi
```

Comment:
That mostly was because I just copied this script from someone else, this probably isnt necessary. Running it twice doesnt seem to affect anything but it is a safeguard indeed